### PR TITLE
Completed project

### PR DESCRIPTION
**Pull request by bx44**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>YouTube video page</title>
```
Comment (MauricioRobayo) [inline index.html]: This is not YouTube 😄 .

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+					
+				</div>
+			</div>
+			<form action="#">
+				<h2>Create an Intuiut account</h2>
+				<p>One account for everything Intuit, including Mint.<br><a href="#">Learn more</a></p>
+				<label for="email">
+					Email address
+				</label>
+					<input type="email" name="email" required>
```
Comment (MauricioRobayo) [inline index.html]: You can add an `id` attribute that matches the `for` attribute of the `label` so they are associated together: https://www.youtube.com/watch?v=HS4bvh_59hU

**Comment by bx44** [comment]
This project I had to do it by myself, because my partner left he program.

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Looking very well, congratulations! 

Just a few minor issues before continuing:

- [ ] Make the README.md more descriptive.
- [ ] Validate `index.html` and fix the `errors`: https://validator.w3.org/#validate_by_input.

[OPTIONAL] Take a look at the `header`, `nav`, `main` and `footer` tags... try to use them instead of `div`s tags where appropriate.

**Status**: Request changes.

**Review by nikasulo — APPROVED** [review]
# Summary
Really nice work! You have one more error to correct courtesy of the HTML validator. I am putting some trust in you that you will correct it before you merge this PR, that being said, I am approving your project.

### See you at the top! :rocket:

**Pull request merged**